packages/payments-plugin/src/mollie/mollie.service.ts

```diff
@@ -1,9 +1,15 @@
-import createMollieClient, { PaymentStatus } from '@mollie/api-client';
+import createMollieClient, {
+    Order as MollieOrder,
+    OrderStatus,
+    PaymentMethod as MollieClientMethod,
+} from '@mollie/api-client';
+import { CreateParameters } from '@mollie/api-client/dist/types/src/binders/orders/parameters';
 import { Inject, Injectable } from '@nestjs/common';
 import {
     ActiveOrderService,
     ChannelService,
     EntityHydrator,
+    ErrorResult,
     LanguageCode,
     Logger,
     Order,
```

Comment:
Add the `OrderStateTransitionError` import here rather than the current deep import.